### PR TITLE
[CHORE] Run Prettier on the other files in the project

### PR DESCRIPTION
**Pull request by j-f1**

### CODE REVIEW

File: docs/rules/member-delimiter-style.md [hunk 1]
```diff
@@ -140,27 +141,27 @@ Examples of **incorrect** code for this rule with the default config:
...
-    greet(): string,
+    name: string;
+    greet(): string;
 }
 
 // missing last member delimiter
 interface Baz {
     name: string;
-    greet(): string
+    greet(): string;
```
Comment (armano2) [inline docs/rules/member-delimiter-style.md]: there should not be last `;`

File: docs/rules/member-delimiter-style.md [hunk 2]
```diff
@@ -140,27 +141,27 @@ Examples of **incorrect** code for this rule with the default config:
...
 }
 
 // incorrect delimiter
-type FooBar = { name: string, greet(): string }
+type FooBar = { name: string; greet(): string };
```
Comment (armano2) [inline docs/rules/member-delimiter-style.md]: incorrect delimiter should be `,`

File: docs/rules/member-delimiter-style.md [hunk 3]
```diff
@@ -140,27 +141,27 @@ Examples of **incorrect** code for this rule with the default config:
...
 
 // last member should not have delimiter
-type FooBar = { name: string; greet(): string; }
+type FooBar = { name: string; greet(): string };
```
Comment (armano2) [inline docs/rules/member-delimiter-style.md]: there should be `;` as last delimiter

**Comment by j-f1** [comment]
Fixed @armano2 ⬆️

**Comment by armano2** [comment]
thank you :>

**Comment by armano2** [comment]
package.json file should use 2 spaces instead of 4

File: package.json [hunk 4]
```diff
@@ -1,59 +1,60 @@
...
-  "husky": {
-    "hooks": {
-      "pre-commit": "yarn check --verify-tree && yarn lint && yarn format-check && yarn test && yarn docs:check && lint-staged"
-    }
-  },
-  "engines": {
-    "node": ">=6"
-  },
-  "license": "MIT"
+    "name": "eslint-plugin-typescript",
```
Comment (armano2) [inline package.json]: package.json file should use 2 spaces instead of 4

File: tests/.eslintrc.yml [hunk 5]
```diff
@@ -1,4 +1,4 @@
 env:
-  mocha: true
+    mocha: true
 rules:
-  node/no-unpublished-require: off # we’re using devDeps here.
+    node/no-unpublished-require: off # we’re using devDeps here.
```
Comment (armano2) [inline tests/.eslintrc.yml]: base on `.editorconfig` yml should have 2 spaces but its overridden by `.prettierrc`

File: package.json (hunk 4 above)
Comment (j-f1) [reply package.json]: Does it preserve the 4-space indentation if you install something?

Comment (armano2) [reply package.json]: editorconfig:

```
# yarn / npm uses 2 spaces when it dumps the file after a change
[package.json]
indent_size = 2

# yml uses 2 spaces
[*.yml]
indent_size = 2
```

Comment (armano2) [reply package.json]: and entire file is rebuilded/formatted when you install something

Comment (bradzacher) [reply package.json]: the latest versions of yarn/npm I believe do preserve the formatting.
It's hard to force contributors to use >= those versions though.

Keeping it at 2 spaces means it's less likely to cause accidental PR diffs.

Comment (j-f1) [reply package.json]: Fixed!

File: .prettierrc [hunk 6]
```diff
@@ -1,11 +0,0 @@
-{
-    "arrowParens": "avoid",
-    "bracketSpacing": true,
-    "jsxBracketSameLine": true,
-    "proseWrap": "preserve",
-    "semi": true,
-    "singleQuote": false,
```
Comment (armano2) [inline .prettierrc]: we have to preserve this option in .editorconfig as
```
quote_type = double
```

**Comment by j-f1** [comment]
Seems like Prettier doesn’t like the format of the README, but it’s fine on my machine.

**Comment by armano2** [comment]
try regenerating `yarn.lock` and run linting again.

**Comment by bradzacher** [comment]
Sorry @j-f1.

I merged something before this one again and I broke it again.
I should have merged this first...

Please fix it one more time and we can merge this as soon as the test passes.

**Comment by j-f1** [comment]
I’m not at a computer right now — would you be able to run `yarn format`?

**Pull request merged**